docs/en/your-profile-and-preferences/customise-mentions-and-other-messages.md

```diff
@@ -0,0 +1,66 @@
+---
+id: 570
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Customise mentions and other messages
+
+You can customise three different types of notifications: `Personal mentions`, `Global Mentions` and `All other messages`. For each one of them, follow the same procedure and use the table below as a reference.
+
+## Know your notification options
+
+| Option | Description | Works with |
+|:---|:---|:---|
+| Personal metions| Include messages that specifically mention you, or have your name tagged. | [Mobile and desktop] |
```

Comment:
- 'Mentions'
- 'mention you or have your name tagged'
- Are 'mention you' and 'have your name tagged' different things?